### PR TITLE
Update to Gtk3 api, update Copyright and fix style issue in stock.rb

### PR DESCRIPTION
**Pull request by cedlemo**

### CODE REVIEW

**Comment by cedlemo** [comment]
Rebase done.

File: gtk3/sample/misc/stock.rb [hunk 1]
```diff
@@ -2,26 +2,25 @@
...
 window = Gtk::Window.new("Gtk::Stock sample")
-window.signal_connect("destroy"){Gtk.main_quit}
+window.signal_connect("destroy") { Gtk.main_quit }
 
 stocks = Gtk::Stock.constants.sort
 
-image = Gtk::Image.new
-label = Gtk::Label.new
-button = Gtk::Button.new("Click!")
+image = Gtk::Image.new :stock => Gtk::Stock::OK, :size => :dialog
```
Comment (cosmo0920) [inline gtk3/sample/misc/stock.rb]: I think that here is braces needed like [this test](https://github.com/ruby-gnome2/ruby-gnome2/blob/master/gtk3/test/test-gtk-image.rb#L6-L7), thanks.

```diff
-image = Gtk::Image.new :stock => Gtk::Stock::OK, :size => :dialog
+image = Gtk::Image.new(:stock => Gtk::Stock::OK, :size => :dialog)
```

**Comment by cedlemo** [comment]
Done :-)

**Comment by cosmo0920** [comment]
Merci bien. :)

**Comment by kou** [comment]
Thanks!

**Pull request merged**